### DISCUSSION TITLE
Tandoor: latest version

### DISCUSSION THREAD

**Discussion by blowk** [alexbelgium/hassio-addons#1021] [Q&A]
Hi 

I'm currently running ghcr.io/alexbelgium/tandoor_recipes-armv7:1.5.6, but when I go to /system, it's shows me the following systems information.

Current Version: [1.4.12](https://github.com/vabene1111/recipes/releases/tag/1.4.12)
Ref: [f2b3cfb8f01e6bc405b4647ee613cf1922b572b0](https://github.com/vabene1111/recipes/commit/f2b3cfb8f01e6bc405b4647ee613cf1922b572b0)

Shoudn't be the current system version matches the docker version?

**Comment by alexbelgium** [comment]
Hi, indeed the armv7 tag was merged with the latest tag. It should be ok again from next version, thanks for noticing

**Reply by blowk** [reply]
ok, thank you very much for the change.
Do you know how I could force it to test it on my HA RPi system?
I see that it only occurs on 32-bit systems. I will check on how migrating to 64-bit.
